tools/android/packaging/Makefile.in

```diff
@@ -1,6 +1,7 @@
 include ../../depends/Makefile.include
 
 OBJS = libcurl.so \
+  librtmp.so \
```

Comment:
> cp: cannot stat '/home/jenkins/workspace/Android-ARM/build/install/lib/librtmp.so': No such file or directory
Makefile:130: recipe for target 'libs' failed
make[4]: *** [libs] Error 1
CMakeFiles/apk-unsigned.dir/build.make:57: recipe for target 'CMakeFiles/apk-unsigned' failed
make[3]: *** [CMakeFiles/apk-unsigned] Error 2
CMakeFiles/Makefile2:1202: recipe for target 'CMakeFiles/apk-unsigned.dir/all' failed
make[2]: *** [CMakeFiles/apk-unsigned.dir/all] Error 2
CMakeFiles/Makefile2:1209: recipe for target 'CMakeFiles/apk-unsigned.dir/rule' failed
make[1]: *** [CMakeFiles/apk-unsigned.dir/rule] Error 2
Makefile:378: recipe for target 'apk-unsigned' failed
make: *** [apk-unsigned] Error 2
Build step 'Execute shell' marked build as failure